docs/components/dartpad/time_pickers/regular/main.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Demo',
+      debugShowCheckedModeBanner: false,
+      theme: ThemeData(colorScheme: ColorScheme.light()),
```

Comment:
I want to keep color scheme otherwise the time picker looks weird. Since it uses the color scheme for its default colors. The default flutter color scheme (with blue) looks bad. This way it matches the materila.io images

Reply:
SGTM